wrappers/Julia/CoolProp.jl

```diff
@@ -297,7 +297,7 @@ end
 # rhomolar The pointer to the array of molar density
 # hmolar The pointer to the array of molar enthalpy
 # smolar The pointer to the array of molar entropy
-function AbstractState_update_and_common_out{R<:Real}(handle::Clong, input_pair::Clong, value1::Array{R}, value2::Array{R}, length::Real, T::Array{R}, p::Array{R}, rhomolar::Array{R}, hmolar::Array{R}, smolar::Array{R})
+function AbstractState_update_and_common_out{R1<:Real,R2<:Real,F<:AbstractFloat}(handle::Clong, input_pair::Clong, value1::Array{R1}, value2::Array{R2}, length::Real, T::Array{F}, p::Array{F}, rhomolar::Array{F}, hmolar::Array{F}, smolar::Array{F})
```

Comment:
Outputs of type AbstractFloat should gives 32/64 bit compatibility out of the box.
The inputs can be arrays of Integers or Floats (32 or 64), to the choice. Best in term of performance is to use `Cdouble` as no conversion will be needed.